ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/views/PaywallBackground.kt

```diff
@@ -0,0 +1,32 @@
+package com.revenuecat.purchases.ui.revenuecatui.views;
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.BlurredEdgeTreatment
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.draw.blur
+import androidx.compose.ui.unit.dp
+import com.revenuecat.purchases.paywalls.PaywallData
+import com.revenuecat.purchases.ui.revenuecatui.RemoteImage
+import com.revenuecat.purchases.ui.revenuecatui.backgroundUrlString
+import com.revenuecat.purchases.ui.revenuecatui.extensions.conditional
+
+@Composable
+internal fun PaywallBackground(data: PaywallData) {
+    data.backgroundUrlString?.let {
+        RemoteImage(
+            urlString = it,
+            modifier = Modifier
+                .fillMaxSize()
+                .conditional(data.config.blurredBackgroundImage) {
+                    // TODO-PAYWALLS: backwards compatibility for blurring
+                    blur(blurSize, edgeTreatment = BlurredEdgeTreatment.Unbounded)
+                        .alpha(blurAlpha)
+                }
+        )
+    }
+}
+
+private val blurSize = 40.dp
+private const val blurAlpha = 0.7f
```

Comment:
Maybe we can create an object to group these? I was thinking something like:
```
private object BackgroundUIConstants {
   val blurSize = 40.dp
   const val blurAlpha = 0.7f
}
```

Reply:
👍🏻 

I wish components were types so we could put these inside :P